### DISCUSSION TITLE
How to position Controls to the right by default?

### DISCUSSION THREAD

**Discussion by gyenabubakar** [storybookjs/storybook#24902] [Help] [answered]
The controls panel shows at the bottom of the canvas by default. Even when I set it to the right, it resets it to the bottom on page reload. How do I make it so that it's at the right by default and stays there?

![Screenshot 2023-11-19 at 1 58 44 PM](https://github.com/storybookjs/storybook/assets/26672526/04e91a23-9455-4161-b780-8978975c3488)

**Comment by ShaunEvening** [comment, marked as answer]
Hey @gyenabubakar 👋 

It looks like you want to set `panelPosition` to `"right"` in a `.storybook/manager.js` file.
Here's the documentation on how to set this up.
https://storybook.js.org/docs/angular/configure/features-and-behavior
